providers/catalog.go

```diff
@@ -156,4 +157,23 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 			Write:     false,
 		},
 	},
+	
+	// Dropbox configuration
+    Dropbox: {
+        AuthType: Oauth2,
+        BaseURL:  "https://api.dropboxapi.com/2/",
```

Comment:
Users should probably have the freedom to call the v1 of the api, so I would remove `2` from the base URL - what do you think?

Reply:
Sure - please add this information to the dropbox slab if you haven't already. Thanks!